src/components/Header/Header.tsx

```diff
@@ -0,0 +1,19 @@
+export function Header() {
```

Comment:
It is important to follow the same way of creating components everywhere
```suggestion
export const Header = () => {
```